### PR TITLE
feat: Add e2e-test ci yaml

### PR DESCRIPTION
**Pull request by duanmengkk**
#### What type of PR is this?

/kind feature

#### Which issue(s) does this PR fix?

Fixes #
#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by duanmengkk** [comment]
image

@wuyingjun-lucky This image should be public

**Comment by duanmengkk** [comment]
@ONE7live @lvtaococo

File: hack/local-down-clusterlink.sh [hunk 1]
```diff
@@ -7,13 +7,15 @@ set -o pipefail
...
 
 #cluster cluster
 delete_cluster $HOST_CLUSTER_NAME
 delete_cluster $MEMBER1_CLUSTER_NAME
+delete_cluster $MEMBER2_CLUSTER_NAME
 
 
-echo "clusterlink local down success"
\ No newline at end of file
+echo "clusterlink local down success"
```
Comment (hanweisen) [inline hack/local-down-clusterlink.sh:21]: add blank line

Comment (duanmengkk) [reply hack/local-down-clusterlink.sh:21]: it  is already has a new line

**Pull request merged**